docs/tutorials/geomesa-hbase-s3-on-aws.rst

```diff
@@ -110,8 +133,19 @@ To configure GeoMesa, remote into the master node of your new AWS EMR cluster us
 
    $ ssh -i /path/to/key ec2-user@$MASTER
 
-Now, download the GeoMesa HBase distribution, replacing ``${VERSION}`` with the appropriate value or setting the
-$VERSION environment variable.
+Installing GeoMesa
+------------------
+
+Now that you have SSH'd into your master server you can test out your HBase and Hadoop installations by running these
+commands:
+
+.. code-block:: shell
+
+    hbase version
+    hadoop version
+
+If everything looks good, download the GeoMesa HBase distribution, replacing ``${VERSION}`` with the appropriate GeoMesa
+Version (e.g. 1.3.4) or setting the ``VERSION`` environment variable.
 
 .. code-block:: shell
 
```

Comment:
I'd suggest adding the following to the end of this file:
Connecting to External HBase Clusters Backed By S3
---------------------------------------------------

To use a EMR cluster to connect to an existing, external HBase Cluster first follow the above instructions to setup the new cluster and install GeoMesa.

The next step is to obtain the hbase-site.xml for the external HBase Cluster, copy to the new EMR cluster and overwrite the copy in /opt/geomesa-hbase_2.11-${VERSION}/conf/ . At this point you may run the geomesa-hbase command line tools.

If you wish to execute SQL queries using Spark, you must first zip the hbase-site.xml file for the external cluster:

.. code-block:: shell

    zip  hbase-site.xml.zip hbase-site.xml

and copy the zip file to /opt/geomesa/conf/ then add the zipped configuration file to the Spark classpath
  .. code-block:: shell

    $ JARS=file:///opt/geomesa/dist/spark/geomesa-hbase-spark-runtime_2.11-${VERSION}.jar,file:///opt/geomesa/conf/hbase-site.xml.zip

Then start up the Spark shell

.. code-block:: shell

    $ spark-shell --jars $JARS

Reply:
+1